### ISSUE TITLE
[sem spec] infra in Informative references

### ISSUE THREAD

**Issue by domel** [w3c/N3#196]
> [infra]
    [Infra Standard](https://infra.spec.whatwg.org/). Anne van Kesteren; Domenic Denicola. WHATWG. Living Standard. URL: https://infra.spec.whatwg.org/

I don't see any place where infra is used.

**Comment by william-vw** [comment]
@domel it seems this is included by default from the template - I don't think we added it.

**Comment by domel** [comment]
@william-vw 
Right. But I think it should be removed (but I don't know how 😀 ).
